src/key2/BrickFloor.java

```diff
@@ -0,0 +1,10 @@
+package key2;
+
+public class BrickFloor implements Floor {
+
+	@Override
+	public String getRepresentation() {
+		return "Finished building wood floor";
```

Comment:
Its just a represenation of the floor, nothing especial, is for diferentiate from other floors. There i miss with the return, it should return "finishing building brick floor"